e2e/cypress/integration/archived_channel/archive_channel_post_spec.js

```diff
@@ -47,4 +57,71 @@ describe('Archived channels', () => {
         // * RHS text box should not be visible
         cy.get('#reply_textbox').should('not.be.visible');
     });
+
+    it.only('MM-T1722 Can click reply arrow on a post from archived channel, from saved posts list', () => {
+        let testArchivedChannel;
+        let postId;
+        let permalink;
+
```

Comment:
Looks like these can be deleted and just declare the variables near as much as possible to the scope where those are to be used.
```suggestion
```